draft-ietf-secevent-http-push.html

```diff
@@ -586,13 +589,14 @@ <h1 id="rfc.section.2">
 <li>The SET Recipient can parse the SET.</li>
 <li>The SET is authentic (i.e., it was issued by the issuer specified within the SET).  </li>
 <li>The SET Recipient is identified as an intended audience of the SET.  </li>
+<li>The SET issuer is recognized as an issuer that the SET Recipient is willing to receive SETs from (i.e., the issuer is whitelisted by the SET Recipient).  </li>
 </ul>
 
 <p> </p>
 <p id="rfc.section.2.p.3">The mechanisms by which the SET Recipient performs this validation are out of scope for this document. SET parsing and issuer and audience identification are defined in <a href="#RFC8417" class="xref">[RFC8417]</a>.  The mechanism for validating the authenticity of a SET is implementation specific, and may vary depending on the authentication mechanisms in use, and whether the SET is signed and/or encrypted (See <a href="#aa" class="xref">Section 3</a>).  </p>
 <p id="rfc.section.2.p.4">The SET Recipient SHOULD ensure that the SET is persisted in a way that is sufficient to meet the SET Recipient's own reliability requirements, and MUST NOT expect or depend on a SET Transmitter to re-transmit or otherwise make available to the SET Recipient a SET once the SET Recipient acknowledges that it was received successfully.  </p>
 <p id="rfc.section.2.p.5">Once the SET has been validated and persisted, the SET Recipient SHOULD immediately return a response indicating that the SET was successfully delivered. The SET Recipient SHOULD NOT perform extensive business logic that processes the event expressed by the SET prior to sending this response.  Such logic SHOULD be executed asynchronously from delivery, in order to minimize the expense and impact of SET delivery on the SET Transmitter.  </p>
-<p id="rfc.section.2.p.6">The SET Transmitter SHOULD NOT re-transmit a SET, unless the response from the SET Recipient in previous transmissions indicated a potentially recoverable error (such as server unavailability that may be transient, or a decryption failure that may be due to misconfigured keys on the SET Recipient's side).  In the latter case, the SET Transmitter MAY re-transmit a SET, after an appropriate delay to avoid overwhelming the SET Recipient (see <a href="#reliability" class="xref">Section 4</a>).  </p>
+<p id="rfc.section.2.p.6">The SET Transmitter SHOULD NOT re-transmit a SET, unless the response from the SET Recipient in previous transmissions times out or it indicated a potentially recoverable error (such as server unavailability that may be transient, or a decryption failure that may be due to misconfigured keys on the SET Recipient's side).  In the latter case, the SET Transmitter MAY re-transmit a SET, after an appropriate delay to avoid overwhelming the SET Recipient (see <a href="#reliability" class="xref">Section 4</a>).  </p>
```

Comment:
With this change, "In the latter case" feels clunky and ambiguous. Maybe invert the order of the sentences, e.g.,

> The SET Transmitter MAY re-transmit a SET if the responses from previous transmissions timed out or indicated potentially recoverable error (such as server unavailability that may be transient, or a decryption failure that may be due to misconfigured keys on the SET Recipient's side). In all other cases, the SET Transmitter SHOULD NOT re-transmit a SET. The SET Transmitter SHOULD delay retransmission for an appropriate amount of time to avoid overwhelming the SET Recipient (see <a href="#reliability" class="xref">Section 4</a>).